source/application/InputCapture.cpp

```diff
@@ -1,870 +1,1037 @@
-	{{SINGLE_INPUT_DI_KEYBOARD, /*DIK_LSHIFT*/ 0x400 | DIK_4},                ID_FILES_SAVESTATE_4,     L"Save State 4"},
-	{{SINGLE_INPUT_DI_KEYBOARD, /*DIK_LSHIFT*/ 0x400 | DIK_5},                ID_FILES_SAVESTATE_5,     L"Save State 5"},
-	{{SINGLE_INPUT_DI_KEYBOARD, /*DIK_LSHIFT*/ 0x400 | DIK_6},                ID_FILES_SAVESTATE_6,     L"Save State 6"},
-	{{SINGLE_INPUT_DI_KEYBOARD, /*DIK_LSHIFT*/ 0x400 | DIK_7},                ID_FILES_SAVESTATE_7,     L"Save State 7"},
-	{{SINGLE_INPUT_DI_KEYBOARD, /*DIK_LSHIFT*/ 0x400 | DIK_8},                ID_FILES_SAVESTATE_8,     L"Save State 8"},
-	{{SINGLE_INPUT_DI_KEYBOARD, /*DIK_LSHIFT*/ 0x400 | DIK_9},                ID_FILES_SAVESTATE_9,     L"Save State 9"},
-	{{SINGLE_INPUT_DI_KEYBOARD, /*DIK_LSHIFT*/ 0x400 | DIK_0},                ID_FILES_SAVESTATE_10,    L"Save State 10"},
-	{{SINGLE_INPUT_DI_KEYBOARD, /*DIK_LSHIFT*/ 0x400 | DIK_F1},               ID_FILES_SAVESTATE_11,    L"Save State 11"},
-	{{SINGLE_INPUT_DI_KEYBOARD, /*DIK_LSHIFT*/ 0x400 | DIK_F2},               ID_FILES_SAVESTATE_12,    L"Save State 12"},
-	{{SINGLE_INPUT_DI_KEYBOARD, /*DIK_LSHIFT*/ 0x400 | DIK_F3},               ID_FILES_SAVESTATE_13,    L"Save State 13"},
-	{{SINGLE_INPUT_DI_KEYBOARD, /*DIK_LSHIFT*/ 0x400 | DIK_F4},               ID_FILES_SAVESTATE_14,    L"Save State 14"},
-	{{SINGLE_INPUT_DI_KEYBOARD, /*DIK_LSHIFT*/ 0x400 | DIK_F5},               ID_FILES_SAVESTATE_15,    L"Save State 15"},
-	{{SINGLE_INPUT_DI_KEYBOARD, /*DIK_LSHIFT*/ 0x400 | DIK_F6},               ID_FILES_SAVESTATE_16,    L"Save State 16"},
-	{{SINGLE_INPUT_DI_KEYBOARD, /*DIK_LSHIFT*/ 0x400 | DIK_F7},               ID_FILES_SAVESTATE_17,    L"Save State 17"},
-	{{SINGLE_INPUT_DI_KEYBOARD, /*DIK_LSHIFT*/ 0x400 | DIK_F8},               ID_FILES_SAVESTATE_18,    L"Save State 18"},
-	{{SINGLE_INPUT_DI_KEYBOARD, /*DIK_LSHIFT*/ 0x400 | DIK_F9},               ID_FILES_SAVESTATE_19,    L"Save State 19"},
-	{{SINGLE_INPUT_DI_KEYBOARD, /*DIK_LSHIFT*/ 0x400 | DIK_F10},              ID_FILES_SAVESTATE_20,    L"Save State 20"},
-	{{SINGLE_INPUT_DI_KEYBOARD, 0},                                           ID_RAM_SEARCH,            L"Ram Search"},
-	{{SINGLE_INPUT_DI_KEYBOARD, 0},                                           ID_RAM_WATCH,             L"Ram Watch"},
-	{{SINGLE_INPUT_DI_KEYBOARD, 0},                                           ID_INPUT_HOTKEYS,         L"Configure Hotkeys"},
-	{{SINGLE_INPUT_DI_KEYBOARD, 0},                                           ID_INPUT_INPUTS,          L"Configure Game Input"},
-	{{SINGLE_INPUT_DI_KEYBOARD, 0},                                           ID_SWITCH_TO_TASEE,       L"Switch to Game Window"},
-	{{SINGLE_INPUT_DI_KEYBOARD, 0},                                           ID_SWITCH_TO_TASER,       L"Switch to Hourglass Window"}
-};
-
-std::map<SingleInput,SingleInput> InputCapture::inputMapping;
-std::map<SingleInput,WORD> InputCapture::eventMapping;
+struct Event InputCapture::eventList[] = {

```

Comment:
Turn off formatting for this here table.

Reply:
Done.